### PR TITLE
Fix erroneous return in VerifyBatches function

### PR DESCRIPTION
**Pull request by KonradIT**
Reported by @cool-develope : https://github.com/0xPolygonHermez/zkevm-node/blob/develop/ethtxmanager/ethtxmanager.go#L153

### What does this PR do?

The function was returning `nil, nil` when everything goes right.

### Reviewers

Main reviewers:

- @cool-develope 
- @arnaubennassar 
- @ARR552 

Codeowner reviewers:

### CODE REVIEW

File: ethtxmanager/ethtxmanager.go [hunk 1]
```diff
@@ -150,7 +150,10 @@ func (c *Client) VerifyBatches(ctx context.Context, lastVerifiedBatch uint64, fi
 			return nil, fmt.Errorf("tx %s failed, err: %w", tx.Hash(), err)
 		} else {
 			log.Infof("batch verification sent to L1 successfully. Tx hash: %s", tx.Hash())
-			return nil, c.state.WaitVerifiedBatchToBeSynced(ctx, finalBatchNum, c.cfg.WaitTxToBeSynced.Duration)
+			return tx, c.state.WaitVerifiedBatchToBeSynced(ctx, finalBatchNum, c.cfg.WaitTxToBeSynced.Duration)
+			if err != nil {
+				return nil, err
+			}
```
Comment (cool-develope) [inline ethtxmanager/ethtxmanager.go:156]:
```suggestion
```

File: ethtxmanager/ethtxmanager.go [hunk 2]
```diff
@@ -150,7 +150,10 @@ func (c *Client) VerifyBatches(ctx context.Context, lastVerifiedBatch uint64, fi
...
 		}
 	}
 	return tx, nil
```
Comment (kind84) [inline ethtxmanager/ethtxmanager.go:159]: @KonradIT I think that we should return an error here if `tx` is nil.

**Review by kind84 — CHANGES_REQUESTED** [review]

**Review by cool-develope — APPROVED** [review]

File: ethtxmanager/ethtxmanager.go (hunk 2 above)
Comment (kind84) [reply ethtxmanager/ethtxmanager.go]: Fixed here 87320ae2d738102cd969c53cdd88fa0a531c995b

File: aggregator/aggregator.go [hunk 3]
```diff
@@ -211,7 +211,7 @@ func (a *Aggregator) sendFinalProof() {
 				continue
 			}
 
-			log.Infof("Final proof for batches %d-%d verified in transaction %v", proof.BatchNumber, proof.BatchNumberFinal, tx.Hash())
+			log.Infof("Final proof for batches [%d-%d] verified in transaction [%v]", proof.BatchNumber, proof.BatchNumberFinal, tx.Hash().TerminalString())
```
Comment (cool-develope) [inline aggregator/aggregator.go:214]: Do you know why this part is not covered in the test?

**Review by kind84 — APPROVED** [review]

**Pull request merged**